components/menu-dropdown/menu-dropdown.jsx

```diff
@@ -338,6 +338,10 @@ const MenuDropdown = createReactClass({
 		 * An object of CSS styles that are applied to the triggering button.
 		 */
 		style: PropTypes.object,
+		/**
+		 * Write <code>"-1"</code> if you don't want the user to tab to the button.
+		 */
+		tabIndex: PropTypes.string,
```

Comment:
I thought about calling this `triggerTabIndex` to make it more clear.  What do you think?